pkg/translator/loki/logs_to_loki.go

```diff
@@ -0,0 +1,99 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loki // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/loki"
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/grafana/loki/pkg/logproto"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+)
+
+func LogsToLoki(logger *zap.Logger, ld plog.Logs) (pr *logproto.PushRequest) {
```

Comment:
Having `zap.Logger` as a parameter of this method would enforce users of `pkg/translator/loki` to use `go.uber.org/zap`. I am wondering whether we can remove this logger method parameter and make this method return an [error](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14106/files#diff-d47b90a2367b146d22e0a67c51700f7c3908dd0b2b47423a3885f38f7f7f8784R84) as well.  Then we can drop `go.uber.org/zap` of this package all together, since it is not used anywhere else. Alternatively, we can add some generic logger wrapper, but it seems like an overkill.

Reply:
Removing would make sense, I believe